Description: disable annotate-snippets, used only in src/_tutorial (docs?)
Forwarded: not-needed
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -223,7 +223,7 @@
 version = "0.4.0"
 optional = true
 
-[dev-dependencies.annotate-snippets]
+[disabled.dev-dependencies.annotate-snippets]
 version = "0.11.3"
 
 [dev-dependencies.anyhow]
